internal/agent/hooks/hooks_test.go

```diff
@@ -79,11 +80,11 @@ var _ = Describe("Hooks", func() {
 				config.WithCloudInitRunner(cloudInit),
 				config.WithImageExtractor(extractor),
 			)
-			cfg.Config = collector.Config{}
+			cfg.Collector = collector.Config{}
```

Comment:
This is a cosmetic thing but makes this more understandable. We are encrusting the collector config into the config, but we used to leave as is, which means it was merging the config with the collector config and calling it Config.Config which was very confusing. This is now much more clear as its the config.Collector instead of just config.Config